src/chrome/content/rules/AOL.xml

```diff
@@ -190,12 +184,6 @@
 
 	<target host="expapi.oscar.aol.com" />
 
-		<!--	Added by AOL, required for http logout.
-								-->
-		<exclusion pattern="^http://my\.screenname\.aol\.com/_cqr/logout/" />
-
-			<test url="http://my.screenname.aol.com/_cqr/logout/" />
-
```

Comment:
Right now, it appears that the URL http://my.screenname.aol.com/_cqr/logout/ leads to a 404 response error page with a message about "They page you are looking for might have been removed." This error page is encountered regardless of whether the URL is accessed via HTTP or HTTPS. It is possible that the URL was functional in the past and that the URLs for logging out of the AOL service have changed over time. (It might be possible for the URL to become functional in the future, but that does not seem to be necessarily likely.)